addons/telestation/fn_ui.sqf

```diff
@@ -0,0 +1,109 @@
+/* ----------------------------------------------------------------------------
```

Comment:
This is up to us, do you think we should have this header on all functions? Personally I like it but I want this decision to be made as collective.

Reply:
I think for functions that should be called by users there should be a header, but it is not necessary for internal functions